theano/configdefaults.py

```diff
@@ -1238,21 +1252,20 @@ def default_blas_ldflags():
                                           "mk2_rt"]])
 
         # Anaconda
-        if "Anaconda" in sys.version or "Continuum" in sys.version:
-            # If the "mkl-service" conda package (available
-            # through Python package "mkl") is installed and
+        if "Continuum Analytics" in sys.version:
```

Comment:
As dicussed elsewhere, I think it is fine to remove the check here, but I think we should conditionally do the warning on line 1264. I don't think it is good that there is a warning when mkl isn't there and we don't use anaconda.

Does conda-forge have the mkl packages?